tests/playwright/sanity/modules/nested-accordion/nested-accordion.test.js

```diff
@@ -149,6 +150,83 @@ test.describe( 'Nested Accordion @nested-accordion', () => {
 				await expect( nestedAccordionItemTitle ).toHaveCount( await numberOfTitles - 1 );
 				await expect( nestedAccordionItemContent ).toHaveCount( await numberOfContents - 1 );
 			} );
+
+			await test.step( 'Check default state behaviour', async () => {
+				// Check default state -> first item is open
+				await expect( nestedAccordionItemTitle.first() ).toHaveAttribute( 'open', '' );
+
+				const allItems = await nestedAccordionItemTitle.all(),
+					allItemExceptFirst = allItems.slice( 1 );
+
+				for ( const item of allItemExceptFirst ) {
+					await expect( item ).not.toHaveAttribute( 'open', '' );
+				}
+
+				// Check all collapsed state -> all items are closed
+				await editor.openSection( 'section_interactions' );
+				await editor.setSelectControlValue( 'default_state', 'all_collapsed' );
+
+				for ( const item of allItems ) {
+					await expect( item ).not.toHaveAttribute( 'open', '' );
+				}
+
+				// Check manual select of first expand -> first item is open
+				await editor.setSelectControlValue( 'default_state', 'first_expended' );
+				await expect( nestedAccordionItemTitle.first() ).toHaveAttribute( 'open', '' );
+
+				for ( const item of allItemExceptFirst ) {
+					await expect( item ).not.toHaveAttribute( 'open', '' );
+				}
+			} );
+		} );
+
+		test( 'Nested Accordion animation', async ( { page }, testInfo ) => {
+			const wpAdmin = new WpAdminPage( page, testInfo ),
+				editor = await wpAdmin.useElementorCleanPost(),
+				container = await editor.addElement( { elType: 'container' }, 'document' ),
+				frame = editor.getPreviewFrame(),
+				nestedAccordionID = await editor.addWidget( 'nested-accordion', container );
+
+			await test.step( 'Check initial animation', async () => {
+				// Wait for the initial "open" animation to complete
+				await page.waitForTimeout( 1500 );
+
+				const newMaxHeight = await frame.isVisible( '.e-n-accordion-item:first-child > div' );
+
+				// Check that the value has changed.
+				expect( newMaxHeight ).not.toEqual( false );
+			} );
+
+			await editor.closeNavigatorIfOpen();
+			await editor.selectElement( nestedAccordionID );
+
+			await test.step( 'Check closing animation', async () => {
+				await frame.locator( '.e-n-accordion-item:first-child > summary' ).click();
+
+				// Wait for the closing animation to complete
+				await page.waitForTimeout( 1500 );
+
+				// Check the computed height
+				const maxHeightAfterClose = await frame.locator( '.e-n-accordion-item:first-child > .elementor-element' ).evaluate( ( element ) =>
+					window.getComputedStyle( element ).getPropertyValue( 'height' ),
+				);
+
```

Comment:
I think that we can add a step that returns a falsy value, 
E.g,  we know that the animation talked 1500ms to be completely opened/closed
Then maybe we can perform a check and see that the content is not open in 1000ms after the click was performed.